Tests/XCUITests/LoginsTests.swift

```diff
@@ -146,6 +147,7 @@ class LoginTest: BaseTestCase {
         openLoginsSettings()
         XCTAssertTrue(app.staticTexts[domain].exists)
         XCTAssertTrue(app.staticTexts[domainLogin].exists)
+        XCTAssertTrue(app.buttons["Edit"].isHittable)
```

Comment:
Not sure if this is going to fix the test, if the element is not there or ready, just asserting if it is hittable will fail same as trying to click on it. But since we are a bit lost with this test, let's try..